playground/terraform/infrastructure/cloudbuild-manual-setup/README.md

```diff
@@ -48,34 +48,25 @@ $BEAM_GIT/playground/terraform/environment/$ENVIRONMENT_NAME/common.tfvars
 ```
 And put the following:
 ```
-playground_deploy_sa = "playground-deploy"
-playground_update_sa = "playground-helm"
-playground_ci_sa = "playground_ci"
-playground_cd_sa = "playground_cd"
-playground_environment_name = "playground-xxx"
-playground_dns_name = "playground.dev"
-playground_network_name = "playground-xxx"
-playground_subnetwork_name = "playground-xxx"
-playground_gke_name = "playground-xxx"
-image_tag = "playground-xxx"
-docker_repository_name = "playground-xxx"
-playground_region = "europe-west3"
-playground_zone = "europe-west3-c"
+playground_deploy_sa = "DEPLOY_SA_NAME"
+playground_update_sa = "UPDATE_SA_NAME"
+playground_ci_sa = "CI_SA_NAME"
+playground_cd_sa = "CD_SA_NAME"
+project_id = "PROJECT_ID"
+playground_environment_name = "ENVIRONMENT_NAME"
```

Comment:
Triggers not environment specific. Can you set environment-name to "environment" or "env" as default value for substitution and not request it?